capnp/lib/capnp.pyx

```diff
@@ -2681,6 +2838,7 @@ cdef class TwoPartyServer:
         """
         Poll libcapnp library forever (asyncio)
         """
+        # TODO: Should probably be removed
```

Comment:
Yeah, I think I agree with this.
Maybe the best way to do this would be to replace it with an exception that it's been deprecated. And in the doc comment a small snippet of python for the alternative.

I likely used this for early testing of asyncio.

Reply:
Done